binota/bangumi-homepage-calendar.user.js

```diff
@@ -432,9 +444,10 @@ var Application = function() {
   var checkBreakPoint = (lastUpdate) ? (new Date(lastUpdate - NSECS_IN_DAY)) : (new Date(now.getTime() - (NSECS_IN_DAY * (DAYS_IN_YEAR + 1))));
 
   //Check
-  var getPage = function(get, calendar, tag) {
-    for(let i = 1; ; i++) {
-      var page = get(i);
+  var getPage = async function(get, calendar, tag) {
+    for (let i = 1; ; i++) {
+      // wait some time let Ukagaka show the message completely;
+      var page = await Promise.all([get(i), wait(200)]).then(([resultA, noResult]) => resultA);
```

Comment:
```javascript
var page = await wait(200).then(() => get(i));
```
Maybe this is better, I think.. 🤔

Reply:
Let 200ms and the request in parallel is no problem.

Reply:
I see. Maybe I misunderstood that.